test/bats/alibabacloud.bats

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bats
+
+load helpers
+
+WAIT_TIME=120
+SLEEP_TIME=1
+NAMESPACE=kube-system
+POD_NAME=basic-test-mount
+BATS_TEST_DIR=test/bats/tests/alibabacloud
+
+setup() {
+  if [[ -z "${ALIBABACLOUD_ACCESS_KEY}" ]] || [[ -z "${ALIBABACLOUD_ACCESS_SECRET}" ]]; then
+    echo "Error: ram ak/sk is not provided" >&2
+    return 1
+  fi
+}
+
+setup_file() {
+    #Configure aliyun cli profile
+    aliyun configure set --profile akProfile --mode AK --region cn-hongkong --access-key-id ${ALIBABACLOUD_ACCESS_KEY} --access-key-secret ${ALIBABACLOUD_ACCESS_SECRET}
```

Comment:
any specific reason to set `cn-hongkong` as default region?

Reply:
refined to us-west-1 as default region